src/test/java/org/broadinstitute/hellbender/utils/read/ReadUtilsIntegrationTest.java

```diff
@@ -0,0 +1,103 @@
+package org.broadinstitute.hellbender.utils.read;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMFileWriter;
+import htsjdk.samtools.SAMRecord;
+import htsjdk.samtools.SamFiles;
+import htsjdk.samtools.SamReader;
+import htsjdk.samtools.SamReaderFactory;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileSystem;
```

Comment:
Also unused.

Reply:
Removed all unused imports in this file.